SUMMARY.md

```diff
@@ -84,9 +84,7 @@
 * [Wallaroo and Virtualenv](book/appendix/virtualenv.md)
 * [TCP Decoders and Encoders](book/appendix/tcp-decoders-and-encoders.md)
 * [Sending Data over TCP with Giles Sender](book/wallaroo-tools/giles-sender.md)
-* [Receiving Data over TCP with Giles Receiver](book/wallaroo-tools/giles-receiver.md)
```

Comment:
Do we have an alternate section for `data_recever`?

Reply:
At the moment no. Either way I was planning a separate PR for documentation...